src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,73 @@
+package edu.kis.vh.nursery;
+
+/**
+ * @author modyfikacji Damian Golema
+ *
+ */
+public class DefaultCountingOutRhymer 
+{
+	/**
+	 * Maksymalny rozmiar stosu (tablica numbers)
+	 */
+	private static final int MAX_SIZE = 12;
+	/**
+	 * Wartość zwracanego błędy przez funkcie
+	 */
+	private static final int ERROR = -1;
+	
+	/**
+	 * Stos 
+	 */
+	private int[] numbers = new int[MAX_SIZE];
+	
+	/**
+	 * przechowuje rozmiar aktualny rozmiar stosu, ERROR jeśli pusty 
+	 */
+	private int total = ERROR;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)